src/quick.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding utf-8 -*-
+"""Implementation of a quick sort."""
+import random
+import timeit
+
+def quick(input_list):
+    """Quick sort implementation."""
+    if not isinstance(input_list, list):
+        raise TypeError
+
+    if len(input_list) <= 1:
+        return input_list
+
+    pivot = input_list[-1]
```

Comment:
Is this the best or only way to choose a pivot? What led you to choose the last item in the list as the pivot?